prow/config/config_test.go

```diff
@@ -2408,6 +2407,21 @@ in_repo_config:
 				return nil
 			},
 		},
+		{
+			name: "InRepoConfigAllowedClusters respects explicit empty default",
+			prowConfig: `
+in_repo_config:
+  allowed_clusters:
+    "*": []
+`,
+			verify: func(c *Config) error {
+				if len(c.InRepoConfig.AllowedClusters) != 1 ||
```

Comment:
This shouldn't tolerate one only zero or not?

Reply:
The value of `c.InRepoConfig.AllowedClusters` is `map["*"]{}`, did you mean we should explicitly make it nil?

Reply:
ah wait, I just misread the diff and thought the first half of the condition was `c.InRepoConfig.AllowedClusters["*"]) != 1`